README.md

```diff
@@ -38,5 +38,10 @@ Build a fixture
 ```
 export SMITHY_PLUGIN_DIR=build/smithy/source/smithy-ruby
 bundle exec smithy-ruby smith client --gem-name fixture --gem-version 1.0.0 <<< $(cat gems/smithy/spec/fixtures/endpoints/default-values/model.json)
+```
 
+Running RBS validations and tests:
```

Comment:
I think eventually the README should be a readme. `rake -T` already accomplishes this assuming we've added descriptions to these rake tasks.